### PR TITLE
TDAP054-Adding-Task-Through-Redux

### PR DESCRIPTION
**Pull request by amit-ib**
Functionality to add new task using redux

### CODE REVIEW

File: src/redux/todoActions.tsx [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export const addTodo = (data: string) => {
```
Comment (akash-dhande-ib) [inline src/redux/todoActions.tsx:1]: For now it's ok, but I think data would be TodoModal

File: src/redux/todoReducer.tsx [hunk 2]
```diff
@@ -10,6 +11,22 @@ const initialState = {
 //(previousState, action) => newState
 const todoReducer = (state: todoList = initialState, action: any) => {
   switch (action.type) {
+    case "ADD_TODO":
+      const updatedTodos = [...state.tasks];
+      if (action.payload) {
+        const todo = {
+          id: state.tasks.length + 1,
```
Comment (akash-dhande-ib) [inline src/redux/todoReducer.tsx:18]: This can make duplicate ids
You can use https://www.npmjs.com/package/uuid

**Review by akash-dhande-ib — APPROVED** [review]

File: src/redux/todoActions.tsx (hunk 1 above)
Comment (amit-ib) [reply src/redux/todoActions.tsx]: Updated and used TodoModal

File: src/redux/todoReducer.tsx (hunk 2 above)
Comment (amit-ib) [reply src/redux/todoReducer.tsx]: getting some issue in installing plugin, will check with you on monday for this

Comment (akash-dhande-ib) [reply src/redux/todoReducer.tsx]: Sure, we will connect for this.

**Pull request merged**